mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -352,6 +355,25 @@ private[clustering] trait LDAParams extends Params with HasFeaturesCol with HasM
       new OldEMLDAOptimizer()
         .setKeepLastCheckpoint($(keepLastCheckpoint))
   }
+
+  def extractTopicDistributionCol(metadata: Metadata, model: Params): Unit =
+  {
```

Comment:
style: brace should go on previous line.  Please compare with existing code when unsure about style